website/docs/d/pod.html.markdown

```diff
@@ -156,6 +156,7 @@ The `option` block supports the following:
 #### Attributes
 
 * `medium` - What type of storage medium should back this directory. The default is "" which means to use the node's default medium. Must be an empty string (default) or Memory. For more info see [Kubernetes reference](http://kubernetes.io/docs/user-guide/volumes#emptydir)
+* `size_limit` - Total amount of local storage required for this EmptyDir volume.
```

Comment:
Would be great to link to the official K8s API description for `Quantity` here. It's sufficiently complex that I'm expecting some confusion (I didn't know the possible formats of it myself).
```suggestion
* `size_limit` - Total amount of local storage required for this EmptyDir volume. [More details here.](https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource?tab=doc#Quantity)